src/utils/history/index.js

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios';
+
+const API_URL = process.env.REACT_APP_API_URL;
+
+export const getHistory = async (token) => {
+  try {
+    let { data } = await axios.get(`${API_URL}/user/history`, {
+      headers: { authorization: token },
+    });
+    if (data.history) return data.history;
+    return [];
+  } catch (err) {
+    throw new Error('Failed to get History.');
+  }
+};
+
+export const setHistory = async (token, video) => {
+  try {
+    let { data } = await axios.post(
+      `${API_URL}/user/history`,
+      { video: video },
+      {
+        headers: { authorization: token },
+      }
+    );
+    if (data.history) return data.history;
+    return [];
+  } catch (err) {
+    console.log('Error: ', err);
+    throw new Error('Failed to add to history.');
+  }
+};
+
+export const deleteHistory = async (token, videoId) => {
+  try {
+    let { data } = await axios.delete(`${API_URL}/user/history/${videoId}`, {
```

Comment:
moake const

Reply:
I'm following `let` at many places, Will raise a different PR for that. Thank you